components/Navbar.jsx

```diff
@@ -1,5 +1,63 @@
 import styled from "styled-components";
+import Link from "next/link";
+import Image from "next/image";
 
 export const Navbar = () => {
-  return <></>;
+  return (
+    <>
+      <nav>
+        <StyledNavContainer>
+          <ul>
+            <li>
+              <Link href="/">
+                <img src="/Icons/MainCueCard-Black.svg" alt="Main Icon" />
+              </Link>
+            </li>
+            <li>
+              <Link href="/createCard">
+                <img src="/Icons/AddCard-Black.svg" alt="Main Icon" />
+              </Link>
+            </li>
+            <li>
+              <Link href="/categories">
+                <img src="/Icons/Categories-Black.svg" alt="Main Icon" />
+              </Link>
+            </li>
+          </ul>
+        </StyledNavContainer>
+      </nav>
+    </>
```

Comment:
You don't need the `<>` fragment here. The `nav` element is the "one" element you`re returning.